### PR TITLE
Remove Google plus 

### PR DESCRIPTION
**Pull request by andrewrady**

### CODE REVIEW

File: _sass/_social.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .twitter     {color: $twitter;}
-.facebook    {color: $facebook;}
-.googleplus  {color: $googleplus;}
+.facebook    {color: $facebook;}\
```
Comment (khalidabuhakmeh) [inline _sass/_social.scss]: what is this `\` about?

File: _includes/head.html [hunk 2]
```diff
@@ -18,11 +18,6 @@
   <meta name="og:description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
   <meta property="og:site_name" content="{{ site.description }}">
 
-  <!-- Social: Google+ / Schema.org  -->
```
Comment (kendaleiv) [inline _includes/head.html]: Do we want to remove this, or just edit the comment?

Comment (andrewrady) [reply _includes/head.html]: probably just have `Schema.org` in the comment

Comment (kendaleiv) [reply _includes/head.html]: I think you still want to include all these `meta` tags under the Schema.org comment.

Comment (khalidabuhakmeh) [reply _includes/head.html]: Yeah, Ken is right. You didn't want to remove those meta tags as they power google search.

Comment (jpvicari) [reply _includes/head.html]: Not really proposing this as a change right now, but at some point should we move away from inline schema and move toward `json-ld`? I think it's cleaner to have everything defined in the json object.

Comment (khalidabuhakmeh) [reply _includes/head.html]: Yeah, we might do that when we move to Hugo

**Pull request merged**